python/cuml/dask/linear_model/base.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from cuml.dask.common import wait_and_raise_from_futures
+from cuml.dask.common.comms import CommsContext
+from cuml.dask.common.input_utils import DistributedDataHandler
+
+
+class BaseLinearModelSyncFitMixin(object):
+
```

Comment:
Quick docstring explaining its role maybe?

Reply:
I'm going to remove this from this PR. I copied this from `cuml/dask/common/base.py` but I think one of the merges might have lost the change I made for the linear models to use it. I'll do this move in a follow-on and add the docs.